specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -84,6 +84,101 @@
           }
         }
       }
+    },
+    "/analyze-conversations/jobs": {
+      "post": {
```

Comment:
Only a suggestion: per [guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations-with-status-monitor), you may support a `GET` for this endpoint to list jobs, or do you think that's not interesting / useful in this scenario. The idea would be that, if a process lost an operation ID, e.g., process crash, it could query for any outstanding operations.

Reply:
we should have a get endpoint.

Reply:
Could be a separate PR if you want since you're still just going into a release branch.

Reply:
This is not part of the current API for analyze-text as well. We have a get endpoint with job Id. However, I will add a backlog to create a list API where we can list all running jobs for the endpoint for the subscription/.